src/arrays.js

```diff
@@ -1,61 +1,74 @@
 const getNthElement = (index, array) => {
-  // your code here
+  let newIndex = index;
+  if (index >= array.length) {
+    newIndex = index - array.length;
+  }
+  return array[newIndex];
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  let csvString = array.join(",");
+  return csvString;
 };
 
 const csvStringToArray = string => {
-  // your code here
+  let anArray = string.split(",");
+  return anArray;
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  return array.concat(element);
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  return array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+  return numbers.map(item => item.toString());
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+  return strings.map(item => item.toUpperCase());
 };
 
 const reverseWordsInArray = strings => {
-  // your code here
+  return strings.map(item => item.split("").reverse().join(""));
 };
 
 const onlyEven = numbers => {
-  // your code here
+  return numbers.filter(item => item % 2 === 0);
 };
 
 const removeNthElement2 = (index, array) => {
-  // your code here
+  const newArray = [...array]; //Make a copy of the original array
+  const modifiedArray = newArray.splice(index, 1); //removes the nth element
+  return newArray; //returns new modified array without removed element
 };
 
 const elementsStartingWithAVowel = strings => {
-  // your code here
+  return strings.filter(item => item.match(/^[aeiou]/i));
 };
 
 const removeSpaces = string => {
-  // your code here
+  let array= string.split(" ")
+  array.filter(item => item.trim());
+  string = array.join("");
+  return string;
 };
 
 const sumNumbers = numbers => {
-  // your code here
+  let sum = 0;
+  numbers.forEach(item => sum += item);
```

Comment:
This line of code I get an ESlint error for: _Arrow function should not return assignment._
Which could be fixed, by using {} around: sum += item.
As an other option for this function, you could look at the reduce array method :)